### PR TITLE
Create dummy.txt

### PR DESCRIPTION
**Pull request by m1trix**

### CODE REVIEW

File: dummy.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Fork you!
```
Comment (m1trix) [inline dummy.txt:1]: Ofcourse I can write comments, silly!

**Comment by enchobelezirev** [comment]
ajfheuighuihg

File: dummy.txt (hunk 1 above)
Comment (enchobelezirev) [inline dummy.txt:1]: askjdasdasdasdxczxca

**Review by m1trix — COMMENTED** [review]
Don't mege, ples.

File: dummy.txt (hunk 1 above)
Comment (m1trix) [reply dummy.txt:1]: But... why?!

**Comment by miroslavLalev** [comment]
sori motori

**Pull request closed without merging**